app/src/main/java/com/cdio/solitaire/model/Card.kt

```diff
@@ -0,0 +1,46 @@
+package com.cdio.solitaire.model
+
+enum class Suit {
+    NA, CLUBS, DIAMONDS, HEARTS, SPADES; // Note the order: C = 1, D = 2, H = 3, S = 4.
+
+    fun short(): String {
+        return when (this.ordinal) {
+            1 -> "C"
+            2 -> "D"
+            3 -> "H"
+            4 -> "S"
+            else -> "NA"
```

Comment:
Consider looking at the enum value directly, instead of the ordinal value. Like so:
```suggestion
        return when (this) {
            CLUBS -> "C"
            DIAMONDS -> "D"
            HEARTS -> "H"
            SPADES -> "S"
            NA -> "NA"
```